MarsTestAutomation/SpecFlow/SellerProfile.Skills.feature

```diff
@@ -0,0 +1,102 @@
+Feature: SellerProfile_Skills
+As a Seller
+I want to add skills in my Profile Details
+So that
+The people looking my profile can see what skills i know.
+
+#	profile skills add
+
+    @sellerprofiletest
+    Scenario: Add Profile skills with data
+        Given I logged into Trade Skills portal successfully
+        And I click on skills tab
```

Comment:
I would put this as a when step as remember given is a prerequisite (a step to to show that this is what state you should be in before you could execute the test)

Reply:
done